src/Resources/views/CRUD/base_edit_form_macro.html.twig

```diff
@@ -8,20 +8,32 @@
             <div class="{{ form_group.box_class }}">
                 <div class="box-header">
                     <h4 class="box-title">
-                        {{ form_group.label|trans({}, form_group.translation_domain ?: admin.translationDomain) }}
+                        {% if form_group.translation_domain is same as(false) %}
+                            {{ form_group.label }}
+                        {% else %}
+                            {{ form_group.label|trans({}, form_group.translation_domain|default(admin.translationDomain)) }}
+                        {% endif %}
                     </h4>
                 </div>
                 <div class="box-body">
                     <div class="sonata-ba-collapsed-fields">
                         {% if form_group.description %}
-                            <p>{{ form_group.description|trans({}, form_group.translation_domain ?: admin.translationDomain)|raw }}</p>
+                            {% if form_group.translation_domain is same as(false) %}
+                                <p>{{ form_group.description|raw }}</p>
+                            {% else %}
+                                <p>{{ form_group.description|trans({}, form_group.translation_domain|default(admin.translationDomain))|raw }}</p>
+                            {% endif %}
                         {% endif %}
 
                         {% for field_name in form_group.fields|filter(field_name => form[field_name] is defined) %}
                             {{ form_row(form[field_name]) }}
                         {% else %}
                             {% if form_group.empty_message != false %}
-                                <em>{{ form_group.empty_message|trans({}, form_group.empty_message_translation_domain) }}</em>
+                                {% if form_group.translation_domain is same as(false) %}
+                                    <em>{{ form_group.empty_message|trans({}, form_group.empty_message_translation_domain) }}</em>
```

Comment:
Missed this one: Should this be 
```suggestion
                                    <em>{{ form_group.empty_message }}</em>
```

to match the other one's? Or was this done on purpose?

Reply:
Indeed, and the if should be `form_group.empty_message_translation_domain is same as(false)`